server/services/participants.js

```diff
@@ -132,7 +132,7 @@ const createChangeHistory = (participantBody, changes) => {
 const updateParticipant = async (participantInfo) => {
   // The below reduce function unpacks the most recent changes in the history
   // and builds them into an object to be used for the update request
-  const changes = participantInfo.history[0].changes.reduce(
+  const changes = participantInfo.history[0]?.changes.reduce(
```

Comment:
Do we need the changes in this file? If they aren't open bugs or blockers I'd like to avoid changing them, optional chaining can hide other issues

Reply:
Sure, I can remove this.